pep-0637.rst

```diff
@@ -536,6 +581,7 @@ need to be implemented to support the extended call. We propose
 - ``PyObject_SetItemWithKeywords(PyObject *o, PyObject *key, PyObject *value, PyObject *kwargs)``
 - ``PyObject_GetItemWithKeywords(PyObject *o, PyObject *key, PyObject *kwargs)``
 
+<<<<<<< HEAD:pep-0637.rst
```

Comment:
Looks like a poorly resolved merge conflict

Reply:
@JelleZijlstra it has been fixed already. Not sure why one travis is hanging on queued though.

Reply:
Sorry for the noise.

If you click through to https://travis-ci.com/github/python/peps/jobs/391286293 it actually passed. I've seen pretty often that Travis status updates don't go through to GitHub for some reason.

Reply:
I try to kick it again. One sec.